### PR TITLE
feat: Show all experiments in monitor

### PR DESCRIPTION
**Pull request by gaocegege**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/835)

### CODE REVIEW

File: pkg/ui/v1alpha3/backend.go [hunk 1]
```diff
@@ -443,3 +423,27 @@ func (k *KatibUIHandler) AddEditDeleteTemplate(w http.ResponseWriter, r *http.Re
...
+	el, err := k.katibClient.GetExperimentList(namespace)
+	if err != nil {
+		log.Printf("GetExperimentList for NAS failed: %v", err)
+		return nil, err
+	}
+	for _, experiment := range el.Items {
+		if experiment.Spec.NasConfig != nil {
+			experimentLastCondition, err := experiment.GetLastConditionType()
+			if err != nil {
+				log.Printf("GetLastConditionType for HP failed: %v", err)
```
Comment (hougangliu) [inline pkg/ui/v1alpha3/backend.go]: s/HP/NAS

File: pkg/ui/v1alpha3/backend.go [hunk 2]
```diff
@@ -443,3 +423,27 @@ func (k *KatibUIHandler) AddEditDeleteTemplate(w http.ResponseWriter, r *http.Re
 	}
 	w.Write(response)
 }
+
+func (k *KatibUIHandler) getExperimentList(namespace string) ([]JobView, error) {
+	jobs := make([]JobView, 0)
+
+	el, err := k.katibClient.GetExperimentList(namespace)
+	if err != nil {
+		log.Printf("GetExperimentList for NAS failed: %v", err)
```
Comment (hougangliu) [inline pkg/ui/v1alpha3/backend.go]: it seems the experiment  list will include all kinds of experiment, not only NAS

File: pkg/ui/v1alpha3/backend.go [hunk 3]
```diff
@@ -80,31 +61,31 @@ func (k *KatibUIHandler) FetchHPJobs(w http.ResponseWriter, r *http.Request) {
...
+	w.Write(response)
+}
+
 func (k *KatibUIHandler) FetchNASJobs(w http.ResponseWriter, r *http.Request) {
 	//enableCors(&w)
-
-	jobs := make([]JobView, 0)
-
-	el, err := k.katibClient.GetExperimentList()
+	jobs, err := k.getExperimentList(consts.DefaultKatibNamespace)
```
Comment (hougangliu) [inline pkg/ui/v1alpha3/backend.go]: why only consts.DefaultKatibNamespace?

File: pkg/ui/v1alpha3/backend.go [hunk 4]
```diff
@@ -80,31 +61,31 @@ func (k *KatibUIHandler) FetchHPJobs(w http.ResponseWriter, r *http.Request) {
...
 
+// FetchAllHPJobs gets experiments in all namespaces.
+func (k *KatibUIHandler) FetchAllHPJobs(w http.ResponseWriter, r *http.Request) {
+	// Use "" to get experiments in all namespaces.
+	jobs, err := k.getExperimentList("")
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
+	log.Printf("?????: %v", jobs)
```
Comment (hougangliu) [inline pkg/ui/v1alpha3/backend.go]: ????? by mistake?

File: pkg/ui/v1alpha3/backend.go [hunk 5]
```diff
@@ -80,31 +61,31 @@ func (k *KatibUIHandler) FetchHPJobs(w http.ResponseWriter, r *http.Request) {
...
+	response, err := json.Marshal(jobs)
+	if err != nil {
+		log.Printf("Marshal NAS jobs failed: %v", err)
```
Comment (hougangliu) [inline pkg/ui/v1alpha3/backend.go]: NAS?

File: pkg/ui/v1alpha3/backend.go (hunk 4 above)
Comment (gaocegege) [reply pkg/ui/v1alpha3/backend.go]: It is still WIP, some bugs are not fixed. Thus it is a WIP log.

Comment (hougangliu) [reply pkg/ui/v1alpha3/backend.go]: Missed WIP tag, please go ahead

**Comment by gaocegege** [comment]
Peek 2019-09-29 17-49

File: pkg/ui/v1alpha3/util.go [hunk 6]
```diff
@@ -1,15 +1,42 @@
...
+	if err != nil {
+		log.Printf("GetExperimentList for NAS failed: %v", err)
+		return nil, err
+	}
+	for _, experiment := range el.Items {
+		if (typ == JobTypeNAS && experiment.Spec.NasConfig != nil) ||
+			(typ == JobTypeHP && experiment.Spec.NasConfig == nil) {
+			experimentLastCondition, err := experiment.GetLastConditionType()
+			if err != nil {
+				log.Printf("GetLastConditionType for NAS failed: %v", err)
```
Comment (johnugeorge) [inline pkg/ui/v1alpha3/util.go]: NAS?

Comment (gaocegege) [reply pkg/ui/v1alpha3/util.go]: Typo, fixed it.

**Pull request merged**